documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -820,6 +826,13 @@ optional https://en.wikipedia.org/wiki/JSON_Web_Token[JSON Web Token (JWT) authe
 |
 |Requested duration (in minutes) before the refresh token expires.
 
+|[[httpclient-authentication-webhook-secret]]<<httpclient-authentication-webhook-secret, `debezium.sink.http.webhook.secret` >>
```

Comment:
@TimoWilhelm  Looking at the code this seems should be `debezium.sink.http.authentication.webhook.secret`. Ins't it?